release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -191,8 +191,15 @@ no operators found with name <name>.<version> in channel <name> of package <name
 === Machine API
 
 
-[id="ocp-4-8-nodes"]
-=== Nodes
+[id="ocp-4-8-mco-upgrade-complete"]
+==== Updates now report status based on all machine config pools
+
+When updating, the Machine Config Operator (MCO) now sets `Upgradeable=False` for any machine config pool that has not completed updating. This status blocks future minor updates, but does not block future patch updates, or the current update. Previously, the MCO reported update progress on the status of the master machine config pools, even if the worker pools were not done updating.   
```

Comment:
@wking Am I any closer here?

Reply:
Based on [this code][1], I expect `Upgradeable=False` when the pool is updating.  But a paused pool may not be updating, even if it "has not completed updating".  I dunno if the MCO logic distinguishes between "paused while level" or "paused while progressing" here, but @kikisdeliveryservice and @yuqi-zhang can probably clear us up.

[1]: https://github.com/openshift/machine-config-operator/pull/2487/files#diff-8a6a064b4db19eb3b29e5280447ac3ba9496e847391489d25c1282767c3354c5R268

Reply:
Paused pools are not presently considered in this process. So "paused while level" == updated (not blocked), and "paused while progressing" == updating (blocked)

Reply:
@mike-nguyen Is this an issue you can review for QE? Thank you!

Reply:
@mike-nguyen PTAL